ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/template2/Template2.kt

```diff
@@ -0,0 +1,160 @@
+package com.revenuecat.purchases.ui.revenuecatui.templates.template2
```

Comment:
Nit, but I wonder if this needs its own folder or it can be inside `templates` instead?

Reply:
Moved to the templates package in #1235